web_src/js/modules/toast.js

```diff
@@ -23,6 +23,8 @@ const levels = {
 async function showToast(message, level, {gravity, position, duration, ...other} = {}) {
   if (!message) return;
 
+  // FIXME: the "toast" mode shouldn't be imported asynchronously.
+  // Otherwise, network errors can't be shown because the toastify itself can't be loaded at that time.
   const {default: Toastify} = await import(/* webpackChunkName: 'toastify' */'toastify-js');
   const {icon, background, duration: levelDuration} = levels[level ?? 'info'];
```

Comment:
Just change it to a sync import above then.

Reply:
Could you help to change this? I am not familiar with webpack async/sync import.

Reply:
Just remove line 28 and the comments above and add this on top of the file:

```js
import Toastify from 'toastify-js';
```

Reply:
Thanks, done in https://github.com/go-gitea/gitea/pull/25719/commits/5c5ac0aaa023b369ed41c11ea2ba21ce630637fa